eis_toolkit/cli.py

```diff
@@ -871,6 +871,43 @@ def local_morans_i_cli(
     typer.echo(f"Local Moran's I completed, output vector saved to {output_vector}.")
 
 
+# FEATURE IMPORTANCE
+@app.command()
+def feature_importance_cli(
+    model_file: INPUT_FILE_OPTION,
+    input_rasters: INPUT_FILES_ARGUMENT,
+    target_labels: INPUT_FILE_OPTION,
+    output_file: OUTPUT_FILE_OPTION,
+    feature_names: Optional[List[str]] = None,
+    n_repeats: int = 50,
+    random_state: Optional[int] = None,
+):
+    """Evaluate the feature importance of a sklearn classifier or regressor."""
+    from eis_toolkit.exploratory_analyses.feature_importance import evaluate_feature_importance
+    from eis_toolkit.prediction.machine_learning_general import load_model, prepare_data_for_ml
+
+    typer.echo("Progress: 10%")
+
+    model = load_model(model_file)
+    typer.echo("Progress: 20%")
+
+    X, y, _, _ = prepare_data_for_ml(input_rasters, target_labels)
+    typer.echo("Progress: 30%")
+
+    feature_importance, result = evaluate_feature_importance(model, X, y, feature_names, n_repeats, random_state)
+    typer.echo("Progress: 80%")
+
+    # np.ndarrays are not JSON serializable -> convert to lists
+    result = {key: value.tolist() for key, value in result.items()}
+
+    feature_importance.to_csv(output_file)
+    json_str = json.dumps(result)
```

Comment:
I don't think we need to give plugin users the whole result dictionary. I think that it suffices to give the mean importances matched with the associated feature name, so what is now returned as a DF. If we want to follow the convention that is now used in EIS Toolkit. a dictionary would be the most suitable form. So a dictionary where keys are feature names and values are the mean importances. However, I've been pondering for some time should all the CLI functions be changed to return files instead of dictionaries. Perhaps best of both worlds would be to optionally return files if it is possible to output file optional in QGIS, I don't remember right now

If you do this modification, remember to remove the `output_file` param